src/myr/Myr.js

```diff
@@ -1047,6 +1053,164 @@ class Myr {
         return outerElId;
     }
 
+    setSeed = (seed = 0) => {
+        if (seed === 0){
+            let newSeed = new Date().getTime();
+            while(this.rand.seed === newSeed)
+            {
+                newSeed = new Date().getTime();
+            }
+            this.rand.seed = newSeed;
+        }
+        else{
+            this.rand.seed = seed;
+        }
+        this.rand.randCounter = 0;
+    }
+
+    getSeed = () => {
+        return this.rand.seed;
+    }
+
+    getSeedCounter = () => {
+        return this.rand.randCounter;
+    }
+
+    setSeedCounter = (counter = 0) => {
+        this.rand.randCounter = counter;
+    }
+
+    decrementRandCounter = () => {
+        this.rand.randCounter -= 1;
+    }
+
+    calculateRandom = (seedNum, nextRand) => {
+        let small = seedNum*0.000000000000000001;
+        let counterPow = this.rand.randCounter * this.rand.randCounter;
+        return (seedNum*Math.PI*counterPow*small*1.543637*9.479137*0.5*nextRand);
+    }
+
+    randomInt = (min = -40, max = 40) => {
+        if(max < min)
+        {
+            let temp = max;
+            min = max;
+            max = temp;
+        }
+        if(max === min)
+        {
+            return max;
+        }
+
+        let range = max - min;
+        let randNum, seedNum, nextRand;
+
+        if(this.rand.seed === 0)
+        {
+            this.rand.seed = new Date().getTime();
+            this.rand.randCounter = 0;
+        }
+
+        if(this.rand.seed !== 0)
```

Comment:
This if is not needed. Your check above already made sure that the seed is not 0.